config_example.json

```diff
@@ -15,19 +15,146 @@
         "fennec_nightly_repacks": "/path/to/beetmoverscript/beetmoverscript/templates/fennec_nightly_repacks.yml",
         "fennecx86_nightly": "/path/to/beetmoverscript/beetmoverscript/templates/fennecx86_nightly.yml"
     },
+
     "s3": {
         "firefox": {
-            "bucket": "mozilla-releng-beetmoverscript-firefox",
-            "credentials": {
-                "id": "dummy",
-                "key": "dummy"
+            "nightly": {
+                "push-to-nightly": {
+                    "s3_prefixes": {
+                        "default": {
+                            "s3_prefix": "pub/firefox/nightly/{}-{}/",
+                            "s3_prefix_latest": "pub/firefox/nightly/latest-{}/"
+                        },
+                        "repacks": {
+                            "s3_prefix": "pub/firefox/nightly/{}-{}-l10n/",
+                            "s3_prefix_latest": "pub/firefox/nightly/latest-{}-l10n/"
+                        }
+                    },
+                    "bucket": "mozilla-releng-firefox-nightly-bucket",
```

Comment:
Apologies for misunderstanding. My bad - it all makes sense now.
I've stashed my work and started fresh. Thanks for clarifications!